test/php/library/Icinga/Application/NoPasswordPolicyTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Tests\Icinga\Application;
+
+use Icinga\Application\Hook\PasswordPolicyHook;
+use Icinga\Test\BaseTestCase;
+use Icinga\Application\ProvidedHook\NoPasswordPolicy;
+
+class NoPasswordPolicyTest extends BaseTestCase
+{
+    private PasswordPolicyHook $instance;
+
+    public function setUp(): void
+    {
+        $this->instance = new NoPasswordPolicy();
+    }
+
+    public function testMethodGetName(): void
+    {
+        $this->assertSame('None', $this->instance->getName());
+    }
+
+    public function testValidatePasswordValid(): void
```

Comment:
Suggestion: you can name the method something like:
`testMethodValidatePasswordAlwaysReturnAnEmptyArray()`